src/reference-guide/controller-api/v2.3.md

```diff
@@ -576,6 +576,67 @@ DEIS_API_VERSION: 2.3
 DEIS_PLATFORM_VERSION: 2.3.0
 ```
 
+### Enable or disable TLS
+
+Example Request:
+
+```
+POST /v2/apps/example-go/tls/ HTTP/1.1
+Host: deis.example.com
+Content-Type: application/json
+Authorization: token abc123
+
+{
+  "https_enabled": true
+}
+```
+
+Example Response:
+
+```
+HTTP/1.1 201 CREATED
+DEIS_API_VERSION: 2.3
+DEIS_PLATFORM_VERSION: 2.3.0
+Content-Type: application/json
+
+{
+    "created": "2014-01-01T00:00:00UTC",
+    "app": "example-go",
+    "owner": "test",
+    "https_enforced": true,
+    "updated": "2014-01-01T00:00:00UTC",
+    "uuid": "de1bf5b5-4a72-4f94-a10c-d2a3741cdf75"
+}
+```
+
+### Get TLS status
+
+Example Request:
+
+```
+GET /v2/apps/example-go/tls/ HTTP/1.1
+Host: deis.example.com
+Authorization: token abc123
+```
+
+Example Response:
+
+```
+HTTP/1.1 200 OK
+DEIS_API_VERSION: 2.3
+DEIS_PLATFORM_VERSION: 2.3.0
+Content-Type: application/json
+
+{
+    "created": "2014-01-01T00:00:00UTC",
+    "app": "example-go",
+    "owner": "test",
+    "https_enforced": false,
```

Comment:
When I create a `foo` app, I'm seeing `"https_enforced": null`:
```
$ deis create foo --no-remote
Creating Application... done, created foo
If you want to add a git remote for this app later, use `deis git:remote -a foo`

$ curl ${CONTROLLER_HOST}/v2/apps/foo/tls/ -H "Authorization: token ${AUTH_TOKEN}" -H "Content-Type: application/json"
{"uuid":"c3abc1da-2b22-4ac7-8132-27574ced2332","app":"foo","owner":"v","created":"2017-01-27T17:49:03Z","updated":"2017-01-27T17:49:03Z","https_enforced":null}
```

(Using deis cli v2.10.0 and latest pre-release v2.10.1 dev chart)